92/terry-iterative.py

```diff
@@ -0,0 +1,27 @@
+# Definition for singly-linked list.
+# class ListNode(object):
+#     def __init__(self, val=0, next=None):
+#         self.val = val
+#         self.next = next
+class Solution(object):
```

Comment:
Recursion is much easier, but sometimes you will be asked to use iteration to solve problems. Nice.